doc/source/api-key-creation.rst

```diff
@@ -0,0 +1,37 @@
+.. _api-key-creation:
+
+====================
+API Key Creation
+====================
+
+API keys are used to authenticate requests made using the Conduce REST API.  API keys are created by the user and should be protected like passwords.  Anyone with an API key can make requests to Conduce and perform operations as the user who requested the key.
+
+Generally, there are three methods used to authenticate Conduce REST requests:
+
+**username/password**
+    used to establish a session
+**session cookie**
+    returned from a user name password login.  Used to continue a session started with a username/password login.
+**API key**
+    a unique string generated upon request that can be used to authenticate requests.
+
+In order to generate an API key, you must have a Conduce account.  Contact support@conduce.com if you do not already have an account.  Once your account is registered, you may create an API key.  It is easiest to do this using the Conduce command line utility.  Once you have :ref:`installed the Conduce Python API <https://github.com/ConduceInc/conduce-python-api>` execute the following command from your shell prompt::
+
+    conduce-api create-api-key --user=new_user@conduce.com --host=app.conduce.com
+
+After you submit your password, the new API key will be printed to the prompt and the command will finish.  Alternatively you may run a Python REPL and call :py:func:`api.create_api_key` directly.::
```

Comment:
You should tell the reader the they will be prompted for a password.  You mention it below with "again", but you only talk about submitting here.

Reply:
I was trying not to be overly wordy.  I was hoping that the prompt would be implied by submission.

I guess since I mention prompting below, I should move that language here and skip it there.

Reply:
yeah... you can't say "again" below if you didn't mention it here first.  And this is the first place they will encounter this, so you should be more descriptive on the initial one